guides/source/active_record_basics.md

```diff
@@ -169,7 +169,9 @@ Rails application with a legacy database? No problem, you can easily override
 the default conventions.
 
 `ApplicationRecord` inherits from `ActiveRecord::Base`, which defines a
-number of helpful methods. You can use the `ActiveRecord::Base.table_name=`
+number of helpful methods. This class should be generated for you automatically
+when creating the application, and lives in `app/models/application_record.rb`.
+You can use the `ActiveRecord::Base.table_name=`
```

Comment:
I like how the current text takes a concept that was introduced previously (`ApplicationRecord`), and uses it to introduce a new one (`ActiveRecord::Base`) and what it provides. Adding this info about `ApplicationRecord` here feels like a disruption in that flow to me. 

What would you think about moving this extra information up to section 3 where `ApplicationRecord` is first introduced (and maybe we should expand on what exactly `ApplicationRecord` does)?

Reply:
@skipkayhil Hey, thanks for the review! My goal here was to answer the question I had which was, "What _is_ `ApplicationRecord` and how did it get there?"

Actually this thought never occurred to me, and my first instinct to search the API docs turned up nothing, because that class doesn't exist until you create it! 😆 

I'm open to ideas on where that fits into this guide, but felt it should be covered.

Feel free to drop a commit on this branch and we can squash+co-sign it! :bow:

Reply:
I think this documentation is confusing because `ApplicationRecord` is an abstract class which means it's not backed by a table. So including the `table_name=` setter in the same short paragraph as `table_name=` seems odd. I know it was like this before but I think this should be reworded more.

Reply:
I don't think I can push to the PR because I'm not a committer, but I pushed a commit to my fork here: https://github.com/rails/rails/commit/278982402b3dab71346c6e0d178b189a2a43bc99

I think this addresses the `ApplicationRecord` feedback by explaining what it does when its first mentioned, and rewording the `ActiveRecord::Base` bit slightly to ensure that its clear that were talking about the application's models.

Reply:
Applied the changes in [e18a091](https://github.com/rails/rails/pull/46700/commits/e18a0911bf7551f4c940fcbb8af9d0bc36197451) if we could get another :eyes: